ansible/roles/common/files/files/authorized_keys

```diff
@@ -0,0 +1,6 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDKCC4nz/iYnqezEX/MfsuWTHJjdR89xjnPq9dPmQCHsAu7/SlKSeY2wAJ3wBgnYNInoXilMN49yAw8WAWgswKibLzqRlci401G/W49RuU7waz2aEGWVnFudm9U7cDqepsJElaQrRLbV1R5OagWc2BHi1XZFxyzs0aPBOsJCMEujfmsY0T85ZzflY4CZ7pNUTvoa83oKYwzk43YDr6tEl2BMSlTZk+y3Rg/Q6UQHZI9m/mm2xyv3/SRbI62zIlEilb/whFMoEb/FIld7DWSO4axmRM1inFFw6d8pzj3eFF7EC/oVUXFA5b1dxU75jy0bw/2hxhaz1o4Crjs+7GpXvGV bsamson@riley.local
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC3CT5adWYD2rYibFwQpB3cbRhc1BeOOjbioB/453beRhATp/183QOMNSS277AlQ8LynF6ZNcHfcRfDzD5vo6XuQfRUY0hBIcnBZJKKOFTeJzOymGS/BaQVxoz1spSZsu6DNL6lVbHhMwM+CMNuT5moPXHiIe9eBbIUkkeThkV2/uoegrAUcOYmdmkIJXLGvp7gYHmH+/7k0lR5sfi1XbZ8OB/xCphcewRZ167/gCbC6NRXABz3W9eRrfDO7jJp+fj2m4JsjPUBBzPn2Spxz3HjgcUEQ98raPvseg33+ju7Sg7fPCp2td4axgBeAe577YwwmMVhHZBx2Lys/IcUsWKH brettsamson@gmail.com
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCqMG+T0g3q/QN0V6MOYcQfMm5pxjWtploLNQ7JMrAuROPo41GZRhV9jyOax0CuCuFvoz+so8fhAAZwd64Yfuq0x4YJUL3SFz27OlvstQYbbNAs6cWP1RwR7BRXA1nAobkJ1UEyakWw8IJHbr8CiacNYn6JY0M/cHHgpfH+OXBp/c5YsaaMIEIybS5tm3IVW4AZjzKCL+4WK7+b163wYQX7T7pUwXjX6ktuGtPQhQflarxoPdYom1QcuPfM24KKQt+1b9EtPVU4bTbfbVQXYrZYKPaIjRPn50o0aOPQeDoFAHGZOFqHLZsKzkQgEeSxuBUueqq8sJ0mP7DiTxvIehE/ zach@crunchbang
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDRd5aOWF/41WswCMmyVnQTSYRGSYGtVJEAauhoV3w3Vcfwr8r8GUwrUvQ5lFvzMqVTzqSWn4S6dVzt+cej0gxdQy40SNuXGRC+3W88tpPjSt08O3AzbaEkQMs19FRfmZ2yGP/WHS/YtA6+ArSnT1sAfWZ9u7UJuX0OBJZF16col2vnZK4FBMT6JSrnjxzJauaSNlLssp8AhCnanL4JfborDwU6To2U3XrANS2m792IXOAzdDBZJVASf7FWV6u3lQNuY7f6cyG0JmDmEFC2yqH+/kwhXfgR1eF7SFBUE3puuUj6e1toZc5ZkAgJ9Hvm3YXVdUOZ0GIGwWMSxJvTWU5T hilarysk@gmail.com
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDXctzutKtkSAjRZ4z50m7HPNNKFc9964WxpX7Sybs8SjgoM88K+P1OwaBDmX1YUpBab/lqKAuuDdeznIap0bopGCIYVlFGBMxf0csXv+Jp2ItrQhNPkCDyserI9W0YNCSAwJpYpaVQqxmorZafoioMb/fK7bGZ+mJ0mMRLvuuLcd34F98PI1eV78Xdu8w4MF5eRLibiTODrRwM29XB+Z52+jPCWSIY5DE1b+BRBtwMxMW2mw4KNdsiLhFT5VDiaYnAnZKyGw5gU8AJ0RHBkuYCTaXxx3LO8Ys1ttKY2I2R8DRO4swekC44NKJveWVbtovdGkWVYzlCJ6x/mRLrV8mzWwPOnhy3U5sYZr9+I3GXBhpsZHpvdM/YH7qhC3wrNcFupy6bjoeO5tE2rCEfM+PariZmbUgP0bJ89kq/5qj8lNu1rtp+w/2gKnZqoxdCZyKWvyR9z82Ico9nkxF03xJUummHzlemM3WtfUyw9Ua5zgFYWwr0zavj8kMrPx3+zE9+CaG7eCtkRYDHcX5RFh/wzrwjo3/kfKBn8+5XMpINegF0U0xlph5wC0OIJB2do+Nzr47GHa1Ity2Tan0WGBacWXBOQ0YiayE/h/4DhNjv00EbBVA7MV8xb/I7xIuWsSJgEHr8xReccTsMxYJZECx1hQBtgMiBx/Lk400t+W+Ovw== alex@tenforwardconsulting.com
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDNMfjXqD+TnTE+1fvjGZ05iEQYq121/Zwts0V+/M3jUvMHcQWbunQVASeofQZ7TAOgwfyNALhCf6dhxkxL9yMb4/ruEqOmcgvZiwWQxBepGe1nO/ElVRrocXj8qY4emmIsyLirVCTY2nf2IpSZmciIMrVbJ5xJYs7fL5+6by/cQkgYvCCQ4BFVzjNz21TkHZ0kEQdryWSH28ZFeFKPQ16xVp5npJ/p1kFbUyfCOQglU3HXSgs5Vp/HqKDhooJ+H/dMvOAipdSCxnHSiOwwABloO7HnSMyilqRaClXuKVej7KL0bLnzVSwc9GGp6lpmMSpewUvu78gTREvw3f9/e5q5 mark@mark.local
```

Comment:
We shouldn't check this in here, we should put this in a .subspace/authorized_keys or something